boxtree/tools.py

```diff
@@ -829,4 +1045,85 @@ def finalize_potentials(self, potentials):
 
 # }}}
 
+
+# {{{ MPI launcher
+
+def run_mpi(script, num_processes, env):
```

Comment:
Someone in the new center made a package for this purpose: https://github.com/majosm/squirm/blob/master/examples/hello_world_call.py
One benefit is that you can pass along Python objects. (such as arg tuples)